lib/WorseReflection/Bridge/TolerantParser/Diagnostics/UnusedImportProvider.php

```diff
@@ -462,18 +469,24 @@ public function name(): string
         return 'unused_import';
     }
 
-    private function extractDocblockNames(Docblock $docblock, NodeContextResolver $resolver, Node $node): void
+    private function extractDocblockNames(DocBlock $docblock, NodeContextResolver $resolver, Node $node): void
```

Comment:
This re-implementation of extracting docblock types relies on parsed types, instead of directly on docblock AST, yet it may pose issues with native parser